### ISSUE TITLE
deploy release fails with: No release_or_upgrade_with_release(s) found in local release store.

### ISSUE THREAD

**Issue by spscream** [boldpoker/edeliver#45]
I have a new phoenix app and I'm trying to setup edeliver for it

Now It successfully builds a release:
```
BUILDING RELEASE OF AUTHORIZATION_AND_ROUTING APP ON BUILD HOST

-----> Authorizing hosts
-----> Ensuring hosts are ready to accept git pushes
-----> Pushing new commits with git to: amalaev@128.199.33.132
-----> Resetting remote hosts to d562d4a2ee57294f9072beed709af27c9eaacf62
-----> Cleaning generated files from last build
-----> Fetching / Updating dependencies
-----> Compiling sources
-----> Detecting exrm version
-----> Generating release
-----> Copying release 0.0.2 to local release store
-----> Copying authorization_and_routing.tar.gz to release store

RELEASE BUILD OF AUTHORIZATION_AND_ROUTING WAS SUCCESSFUL!
```

But on deploy stage it fails:
```
$ mix edeliver deploy release to production        

DEPLOYING RELEASE OF AUTHORIZATION_AND_ROUTING APP TO PRODUCTION HOSTS

-----> Authorizing hosts
ls: .deliver/releases/authorization_and_routing_*.release_or_upgrade_with_release.tar.gz: No such file or directory
No release_or_upgrade_with_release(s) found in local release store.
You can build one with the ./edeliver build release_or_upgrade_with_release task.
```

there are release files inside of .deliver/releases, but edeliver detects wrong release type
```
$ ls -l .deliver/releases
total 98352
-rw-rw-r--  1 amalaev  staff  25159494  2 фев 12:33 authorization_and_routing_0.0.1.release.tar.gz
-rw-rw-r--  1 amalaev  staff  25193615  2 фев 14:07 authorization_and_routing_0.0.2.release.tar.gz
```

**Comment by bharendt** [comment]
This looks like a bug. I will take a look at it. For now you could deploy your release by passing the version as command line argument like `--version=0.0.2´. That should work.

**Comment by bharendt** [comment]
Thanks for reporting this. We published the fix as version [1.1.4](https://hex.pm/packages/edeliver/1.1.4).

**Comment by spscream** [comment]
That works! Thank you!
